ci-operator/step-registry/ipi/conf/manual-creds/remove-unnecessary-creds/ipi-conf-manual-creds-remove-unnecessary-creds-commands.sh

```diff
@@ -77,11 +84,21 @@ echo "Baseline Capability Set: $base_operators"
 all_caps=$(echo "$base_operators $ADDITIONAL_ENABLED_CAPABILITY_SET" | xargs -n1 | sort -u | xargs)
 echo "Enabled Capability Set: $all_caps"
 
-if [[ "${ocp_version}" == "4.12" ]]; then
-    if [[ ! "${all_caps}" =~ "Storage" ]]; then
-        namespace="openshift-cluster-csi-drivers"
-        remove_secrets "${SHARED_DIR}" "${namespace}" || exit 1
-    fi
-fi
+case ${ocp_version} in
+"4.12")
+  if [[ ! "${all_caps}" =~ "Storage" ]]; then
+      namespace="openshift-cluster-csi-drivers"
+      remove_secrets "${SHARED_DIR}" "${namespace}" || exit 1
+  fi
+  ;;
+"4.13")
```

Comment:
Is there other solution, that can help us not have to change the code when a new version come out.

Reply:
updated.